app/src/main/res/menu/menu_channel.xml

```diff
@@ -17,6 +17,13 @@
         android:title="@string/share"
         app:showAsAction="ifRoom" />
 
+    <item
+        android:id="@+id/menu_item_notify"
+        android:checkable="true"
+        android:visible="false"
+        android:title="@string/get_notified"
+        app:showAsAction="never" />
```

Comment:
Maybe use the bell icon and make this shown `ifRoom`?

Reply:
We already have 2 icons on toolbar: share and rss, the third one is excess, I think

Reply:
Well, let Android decide if there are two many icons. `ifRoom` does exactly that. What's the pros of hiding an option behind a menu if there is space for it?